app/src/main/java/com/example/mylittlelibrary/ui/activities/MainActivity.kt

```diff
@@ -14,7 +14,7 @@ class MainActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        //testing
+        // testing
```

Comment:
A space before the comment's contents would be more in-line with kotlinlang.org's coding conventions.
https://kotlinlang.org/docs/coding-conventions.html#horizontal-whitespace

Reply:
Is this comment needed at all?